examples/default.yaml

```diff
@@ -244,6 +244,12 @@ firmware:
   # 🟢 Builtin default: false
   legacyBIOS: null
 
+audio:
+  # QEMU audiodev, e.g., "none", "coreaudio", "pa", "alsa", "oss".
+  # Choosing "none" will mute the audio output, and not play any sound.
+  # 🟢 Builtin default: ""
```

Comment:
The default should be "none"

Reply:
It was mostly to make it backwards-compatible, the "" means that it won't have any device at all

```
       -audiodev none,id=id[,prop[=value][,...]]
           Creates a dummy backend that discards all outputs.  This backend
           has no backend specific properties.
```

Maybe it could have been documented better